lib/tasks/outrigger.rake

```diff
@@ -3,7 +3,11 @@ namespace :db do
     desc 'Run migrations for a Tag'
     task tagged: %i[environment load_config] do |_t, args|
       puts("Migrating Tags: #{args.extras}")
-      ActiveRecord::Migrator.migrate(ActiveRecord::Migrator.migrations_paths, &Outrigger.filter(args.extras))
+      if ActiveRecord::VERSION::MAJOR == 5 && ActiveRecord::VERSION::MINOR > 1
```

Comment:
The way this check is structured, it means future upgrades to add support for Rails 6.0 will revert back to using the old API. Doesn't look like the MigrationContext API was present prior to 5.2, so I guess that just means this conditional will get a little ugly:

```
if ActiveRecord::VERSION::MAJOR > 5 ||
  (ActiveRecord::VERSION::MAJOR == 5 && ActiveRecord::VERSION::MINOR > 1)
```

Something like that.